templates/taxi/car_detail.html

```diff
@@ -3,20 +3,34 @@
 {% block content %}
   <h1>
     {{ car.model }}
-    <a href="{% url 'taxi:car-delete' pk=car.id %}" class="btn btn-danger link-to-page">
-      Delete
-    </a>
-
-    <a href="{% url 'taxi:car-update' pk=car.id %}" class="btn btn-secondary link-to-page">
-      Update
-    </a>
   </h1>
   <p>Manufacturer: ({{ car.manufacturer.name }}, {{ car.manufacturer.country }})</p>
-  <h1>Drivers</h1>
+  <div class="container-fluid">
+    <div class="row">
+      <h1>Drivers</h1>
+      <div class="col">
+        <form method="post" action="{% url 'taxi:toggle-driver' car.id %}">
+          {% csrf_token %}
+          {% if user in car.drivers.all %}
+            <input type="submit" value="Delete me from this car" class="btn btn-danger link-to-page">
+          {% else %}
+            <input type="submit" value="Assign me to this car" class="btn btn-success link-to-page ">
+          {% endif %}
+        </form>
```

Comment:
Consider adding a condition to ensure that only users with the `Driver` role can see and interact with this form. This will prevent unauthorized users from attempting to toggle their association with the car.